apps/asap-server/test/handlers/invalidate-cache/index-handler.test.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  CloudFrontClient,
+  CreateInvalidationCommand,
+} from '@aws-sdk/client-cloudfront';
+import { createHandler } from '../../../src/handlers/invalidate-cache/index-handler';
+const mockSend = jest.fn();
+
+jest.mock('@aws-sdk/client-cloudfront', () => ({
+  CloudFrontClient: jest.fn().mockImplementation(() => ({
+    send: mockSend,
+  })),
+  CreateInvalidationCommand: jest
+    .fn()
+    .mockImplementation(() => ({ isMock: true })),
+}));
+
+describe('invalidate cache', () => {
+  afterEach(() => jest.clearAllMocks());
+  afterEach(() => jest.useRealTimers());
+
+  test('invalidation is called on cloudfront', async () => {
+    const dateReference = '2021-07-06T09:21:23.000Z';
+    jest
+      .useFakeTimers('modern')
+      .setSystemTime(new Date(dateReference).getTime());
+
+    const distributionId = 'fake-distribution-id';
+    const event = {
+      Records: [],
+    };
+    const handler = createHandler(distributionId);
+    await handler(event);
+    expect(CloudFrontClient).toHaveBeenCalledWith({
```

Comment:
i know there are different versions you can use but i still have a feeling expecting this is like saying 'the code is what it is'

In fact I would be tempted to not jest this handler at all or just a simple sanity check.

Reply:
It's covering that the command has the right object passed to it. The command is created before being sent. 

The test is relatively simple, it's checking the send method is called with the right command.